account_payment_show_invoice/views/account_move_line.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_move_form" model="ir.ui.view">
+            <field name="name">account.move.form (in account_payment_show_invoice)</field>
+            <field name="model">account.move</field>
+            <field name="inherit_id" ref="account.view_move_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='line_ids']/tree/field[@name='name']" position="after">
+                    <field name="payment_invoice_ids"/>
+                </xpath>
+            </field>
+    </record>
+    <record id="view_move_line_tree" model="ir.ui.view">
+            <field name="name">account.move.line.tree (in account_payment_show_invoice)</field>
+            <field name="model">account.move.line</field>
+            <field name="inherit_id" ref="account.view_move_line_tree"/>
+            <field name="arch" type="xml">
+                <field name="move_id" position="after">
+                    <field name="payment_invoice_ids"/>
```

Comment:
@rousseldenis adding a not stored computed field to `account.move.line` list could be dangerous for performance

Reply:
@eLBati In production for months and no performance problem. Mutli-company, thousands of moves. Could you update your review?

Reply:
:+1: